com.unity.multiplayer.mlapi/Runtime/Messaging/MessageQueue/MessageQueueContainer.cs

```diff
@@ -129,7 +127,7 @@ public uint GetStreamBufferFrameCount(MessageQueueHistoryFrame.QueueFrameType qu
                 return null;
             }
 
-            writer = BeginAddQueueItemToFrame(messageType, Time.realtimeSinceStartup, transportChannel, 0,
+            writer = BeginAddQueueItemToFrame(messageType, Time.realtimeSinceStartup, transportChannel, NetworkManager.LocalClientId,
```

Comment:
Was this a bug or?

Reply:
Well, it was a potential bug...
Every call to EnterInternalCommandContext was setting this to be 0 every time (i.e. from the server).  Which when you call 
MessageQueueHistoryFrame.GetCurrentQueueItem this returns a MessageFrameItem that sets that value as the MessageFrameItem .NetworkId.  
So, if we ever fixed non-batched message sending (currently if you disable message batching it is broken for the Host) that value is used as the sender of the message which would result in it always sending from the server and never from clients.  So, I made that change in the event we fix the other issues when sending with message batching turned off.

Most likely most of this will get replaced with the up-and-coming Zero-alloc messaging flow.  It is benign to message batching and only will help reduce the number of issues needing to be fixed if non-batched message sending gets fixed before the Zero-alloc messaging flow stuff gets implemented and merged (then it won't matter either way).